test/cmdlineTests/no_append_metadata/output

```diff
@@ -0,0 +1,4 @@
+
+======= no_append_metadata/input.sol:C =======
+Binary:
+608080604052346013576035908160198239f35b600080fdfe6004361015600c57600080fd5b6000803560e01c6326121ff014602157600080fd5b346032578060031936011260325733ff5b80fd
```

Comment:
Probably not the best way to test this as this could change in PRs that change the codegen some way. Hopefully, the contract is simple enough that this is not so likely in most PRs. The other option is to have like a boost cpp test, and check for the last byte or something similar.

Reply:
Yeah, this is tricky because the bytecode detection works by finding the metadata :) But the upside is that the fact that it does not get replaced means that it works. I'd just add a comment in the input file that this binary blob is actually the expected result here and `<BYTECODE REMOVED>` means failure.

As for codegen changes, I think it's not a problem. We already have a lot of that churn in tests that produce Yul and we live with it. The biggest reason for removing the bytecode was actually the metadata because that would change not just with codegen changes but also with the environment (e.g. force release builds in CI have different compiler version format in the metadata). If it's gone, the problem is gone :)

Reply:
Why not use ``--bin-runtime`` on an empty contract? That'd just be ``600080fd`` and probably never change...

Reply:
Right. That better better than what is there now but still not a great test overall - simply because if it changes when you update tests it might still be hard to realize that it changed the wrong way. On the other hand if you had something like `<BYTECODE REMOVED><METADATA REMOVED>` and the `<METADATA REMOVED>` part disappeared, it would be very clear.

Reply:
Going to use an empty contract, hopefully would be less flaky. Will probably have to use `--via-ir` so that there is none of those free memory initializations :)